pkg/status/controller.go

```diff
@@ -108,20 +104,18 @@ func NewController(wdsRestConfig *rest.Config, imbsRestConfig *rest.Config,
 		imbsDynClient:  imbsDynClient,
 		imbsKubeClient: imbsKubeClient,
 		workqueue:      workqueue.NewRateLimitingQueue(ratelimiter),
-		listers:        listers,
-		informers:      informers,
```

Comment:
The parameters `listers` and `informers` are no longer used in this func and should be removed from the signature. Right?

Reply:
Exactly. Removed in latest push.